python-threatexchange/threatexchange/cli/label_cmd.py

```diff
@@ -157,7 +157,7 @@ def execute(self, settings: CLISettings) -> None:
         self.action(settings)
 
     def execute_upload(self, settings: CLISettings) -> None:
-        api = settings.get_api_for_collab(self.collab)
+        api = settings.apis.get_for_collab(self.collab)
         # signal_types = self.only_signals or settings.get_signal_types_for_content(
```

Comment:
(ignorable) nit: add ToDo on why this comment is still here or remove?

Reply:
The comment is in the class header - the whole command is still under construction